internal/api/oidc/auth_request.go

```diff
@@ -248,3 +260,24 @@ func setContextUserSystem(ctx context.Context) context.Context {
 	}
 	return authz.SetCtxData(ctx, data)
 }
+
+func (o *OPStorage) getOIDCSettings(ctx context.Context) (accessTokenLifetime, idTokenLifetime, refreshTokenIdleExpiration, refreshTokenExpiration time.Duration, err error) {
+	accessTokenLifetime = o.defaultAccessTokenLifetime
+	idTokenLifetime = o.defaultIdTokenLifetime
+	refreshTokenIdleExpiration = o.defaultRefreshTokenIdleExpiration
+	refreshTokenExpiration = o.defaultRefreshTokenExpiration
+
+	oidcSettings, err := o.query.OIDCSettingsByAggID(ctx, authz.GetInstance(ctx).InstanceID())
+	if err != nil && !errors.IsNotFound(err) {
+		err = nil
+		return
+	}
```

Comment:
do i get this correct, we ignore all errors apart from "not found"? is there a reason for this?

Reply:
Maybe a little confusing, the idea was to return the default if any error occurs, but handle "not found" differently, as "not found" is the only error that could be expected here.

Corrected the logic, that errors besides "not found" result in error return, and then it checks if any oidcSettings are returned, if not the default is used.